collections/incidents.coffee

```diff
@@ -67,4 +67,22 @@ if Meteor.isServer
           newIncident.date = incident.date
 
         Incidents.update(incident._id, {$unset: {deaths: ""}})
-        Incidents.insert(newIncident)
\ No newline at end of file
+        Incidents.insert(newIncident)
+
+    # Convert string case and death counts to integers
+    incidents = Incidents.find({$or: [{cases: {$type: "string"}}, {deaths: {$type: "string"}}]}).fetch()
+    for incident in incidents
+      if incident.cases
+        parsed = parseInt(incident.cases)
+        if parsed.toString() is "NaN"
+          mongoProjection = {$set: {specify: incident.cases}, $unset: {cases: ""}}
+        else
+          mongoProjection = {$set: {cases: parsed}}
+      else if incident.deaths
+        parsed = parseInt(incident.deaths)
+        if parsed.toString() is "NaN"
+          mongoProjection = {$set: {specify: incident.deaths}, $unset: {deaths: ""}}
+        else
+          mongoProjection = {$set: {deaths: parsed}}
+      if mongoProjection
```

Comment:
I think the mongoProjection variable form the last iteration might still be defined.